papaparse.js

```diff
@@ -957,6 +957,7 @@
 
 		var self = this;
 		var _stepCounter = 0;	// Number of times step was called (number of rows parsed)
+		var _rowsCounter = 0;	// Total number of rows that have been parsed so far
```

Comment:
I think the total word is not requried on the comment, so it will be: _Number of rows that had been parsed so far_